src/components/Viewport.ts

```diff
@@ -1781,11 +1781,8 @@ export default class Viewport {
       const { addedPanels, removedPanels } = this.checkVisiblePanelChange(prevVisiblePanels, newVisiblePanels);
 
       if (newVisiblePanels.length > 0) {
-        const firstPanelPos = this.panelManager.firstPanel()!.getPosition();
         const firstVisiblePanelPos = newVisiblePanels[0].getPosition();
-        if (firstVisiblePanelPos >= firstPanelPos) {
```

Comment:
@WoodNeck Is always true?

Reply:
Yes, as it sets firstVisiblePanelPos on position 0